### PR TITLE
Add OPA options to Admin Settings

### LINKED ISSUE #6783: Extend Admin Settings by OPA options
Extend Admin Settings by `enable` and `enforce` OPA options

### PR DESCRIPTION
**Pull request by kgroschoff**
**What this PR does / why we need it**:
Added `enable` & `enforce` OPA options to Admin Settings

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #6783

**Does this PR introduce a user-facing change?**:

```release-note
Added `enable` & `enforce` OPA options to Admin Settings
```

### CODE REVIEW

File: pkg/handler/test/helper.go [hunk 1]
```diff
@@ -1048,6 +1048,10 @@ func GenDefaultSettings() *kubermaticv1.KubermaticSetting {
 			DisplayTermsOfService: false,
 			EnableDashboard:       true,
 			EnableOIDCKubeconfig:  false,
+			OpaOptions: kubermaticv1.OpaOptions{
```
Comment (moadqassem) [inline pkg/handler/test/helper.go]: Nit: actually this is not needed to add it here, as the field is not a pointer and the default values are false anyway ;)

File: pkg/provider/kubernetes/settings.go [hunk 2]
```diff
@@ -99,6 +99,10 @@ func (s *SettingsProvider) createDefaultGlobalSettings() (*kubermaticv1.Kubermat
 				MaxRAM:    128,
 				EnableGPU: false,
 			},
+			OpaOptions: kubermaticv1.OpaOptions{
```
Comment (moadqassem) [inline pkg/provider/kubernetes/settings.go]: Nit: actually this is not needed to add it here, as the field is not a pointer and the default values are false anyway ;)

File: pkg/test/e2e/api/settings_test.go [hunk 3]
```diff
@@ -65,6 +65,10 @@ func TestGetDefaultGlobalSettings(t *testing.T) {
 					MaxRAM:    0,
 					EnableGPU: false,
 				},
+				OpaOptions: kubermaticv1.OpaOptions{
```
Comment (moadqassem) [inline pkg/test/e2e/api/settings_test.go]: Nit: actually this is not needed to add it here, as the field is not a pointer and the default values are false anyway ;)

**Review by moadqassem — COMMENTED** [review]
added minor comments

File: pkg/handler/test/helper.go (hunk 1 above)
Comment (kgroschoff) [reply pkg/handler/test/helper.go]: Thanks @moadqassem, fixed it :)

**Comment by kgroschoff** [comment]
@moadqassem  PTAL :)

**Review by moadqassem — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**